docs/deployment/ci-cd.md

```diff
@@ -4,114 +4,154 @@ SPDX-FileCopyrightText: Copyright (C) 2023 Opal Health Informatics Group at the
 SPDX-License-Identifier: Apache-2.0
 -->
 
-_This document refers to some features and uses of GitLab CI/CD which have not yet been converted to GitHub Actions._
-_Content will be updated as this work progresses._
-
 # GitHub Actions
 
 This project is configured with [GitHub Actions](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions)
 to build and deploy the application automatically.
-Currently, deployment is only supported for internal distribution, not to end users via the app stores.
+This includes deployment for internal distribution, as well as to the app stores.
+
+## Quick Guide
+
+### 1. Building the Dev App
+
+The Dev app is automatically built and deployed to Firebase App Distribution when PRs are merged.
+
+### 2. Building the Prod App and Releasing to the Stores
+
+To build and release the prod app to the stores, follow these steps:
+
+  1. Go to https://github.com/opalmedapps/opal-app/actions, select `Build and Deploy App`,
+     and next to the `workflow_dispatch`, select `Run workflow` with the following options:
+      - Deploy the app via the app stores: yes.
+      - A specific build number to use: enter a number — check the stores for the next sequential number to use.
+  2. When the build finishes:
+      - It will have been automatically uploaded to TestFlight (Apple).
+      - You'll need to manually upload the `android-app-release` aab file to Google Play (Android).
+  3. The app won't automatically be published to users;
+     you'll still need to create new releases in the stores and submit them for review.
+
+## Workflow Descriptions
 
-Two different workflows are handled as follows.
+The different workflows are handled as follows.
 
-## Workflows - GitHub
+### CI
 
-The `Build App` workflow runs automatically on all commits to the main branch, to build the app for iOS and Android.
+This workflow executes automatically for all pull-request, merge-queue and main-branch commits,
+to perform a series of checks on the updated code. It can also be run manually. These checks include:
+
+- Linting via pre-commit
+- License checks via REUSE
+- Validation of the contents of THIRDPARTY.md
```

Comment:
We could consider adding links to pre-commit and REUSE, and maybe the thirdparty check action.

But not really necessary :)